src/popup/popup.css

```diff
@@ -60,12 +67,13 @@ button#stopMonitorBtn:hover {
 }
 
 .success {
-  background-color: #12bc00;
+  background-color: rgb(9, 92, 9);
   color: #ffffff;
 }
 
 .failure,
 #errorText {
   background-color: #d70022;
+
```

Comment:
Nit, remove changes unrelated to the issue being fixed, e.g. this new empty line here and in the popup.html (especially if there isn't any particular reason to apply that change). In general the patch diff should be as focused on the issue being fixed (and clean) as possible.